packages/web-components/src/components/content-group/content-group-heading.ts

```diff
@@ -39,8 +50,12 @@ class C4DContentGroupHeading extends StableSelectorMixin(LitElement) {
     super.connectedCallback();
   }
 
+  firstUpdated() {
+    this._renderHeading();
+  }
+
   render() {
-    return html` <slot></slot> `;
+    return html`<h3 part="heading"><slot></slot></h3>`;
```

Comment:
This won't work - the big thing we're trying to do is put the `<h3>` in the light dom rather than the shadow DOM.